pkg/divert/weaver/divert_test.go

```diff
@@ -679,4 +679,55 @@ func Test_divertIngresses(t *testing.T) {
 	if !reflect.DeepEqual(expectedE3, resultE3) {
 		t.Fatalf("Didn't compute e3 correctly: \n%v\n%v", expectedE3, resultE3)
 	}
+
+	// Eliminate elements from the cache to force RCs
+	d.cache.developerIngresses = map[string]*networkingv1.Ingress{}
+	d.cache.developerServices = map[string]*apiv1.Service{}
+	d.cache.developerEndpoints = map[string]*apiv1.Endpoints{}
+	d.Deploy(ctx)
+
+	resultI1, _ = c.NetworkingV1().Ingresses("cindy").Get(ctx, "i1", metav1.GetOptions{})
+	if !reflect.DeepEqual(expectedI1, resultI1) {
```

Comment:
Can we replace all the reflect.DeepEqual with assert.NotEqual? In case it fails it provides a much better error

Reply:
@jLopezbarb  `assert` isn't working for me, `reflect.DeepEqual` checks each value in the whole object, should ` assert.NotEqual` work the same way?

Reply:
assert uses `reflect.DeepEqual` inside, I don't know why it's not working 🤔

Reply:
@jLopezbarb It works! I was using a different `assert` library :-)